doc/project/project_roles.rst

```diff
@@ -170,6 +173,26 @@ the latter is not possible.
 
 .. _release-engineering-team:
 
+Static Analysis Audit Team
```

Comment:
The _release-engineering-team tag should be next to its section.
(maybe this new section should just be below the release engineering team section)

Reply:
good catch !